libhyper.yaml

```diff
@@ -0,0 +1,41 @@
+package:
+  name: libhyper
+  version: 0.14.27
+  epoch: 0
+  description: "An HTTP library for Rust"
+  copyright:
+    - license: MIT
+
+environment:
+  contents:
+    packages:
+      - wolfi-base
+      - busybox
+      - ca-certificates-bundle
+      - build-base
+      - rust
+      - rustup
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/hyperium/hyper
+      tag: v${{package.version}}
+      expected-commit: d77c2599bc023b258b90a17f5b633c8b7b0cbd4b
+
+  - runs: |
+      rustup toolchain add nightly
+      export PATH=$HOME/.rustup/toolchains/nightly-${{host.triplet.rust}}/bin:$PATH
+      RUSTFLAGS="--cfg hyper_unstable_ffi" \
+        cargo rustc --features client,http1,http2,ffi -Z unstable-options \
+        --crate-type cdylib --release
+
+  - runs: |
+      mkdir -p ${{targets.destdir}}/usr/lib
+      mv target/release/libhyper.so ${{targets.destdir}}/usr/lib/
```

Comment:
Shouldn't this have an SONAME or something?

Reply:
You mean like a libhyper.so.0 or something?

Reply:
Yeah, exactly.  And linked with the `-soname` LDFLAG to ensure the dependency name is set up correctly.

Reply:
How's this?